### ISSUE TITLE
Compilation Error on Generic 5.15.x

### ISSUE THREAD

**Issue by marceloiury** [FreedomBen/rtl8188ce-linux-driver#157]
Hello Fellows!

I got the following error when compiling the generic 5.15.x branch.  Any help?

/home/marcelo/Git/rtl8188ce-linux-driver/rtl8188ee/pwrseqcmd.c:70:17: error: implicit declaration of function ‘RT_TRACE’; did you mean ‘RTA_SPACE’? [-Werror=implicit-function-declaration]
   70 |                 RT_TRACE( rtlpriv, COMP_INIT, DBG_TRACE,
      |                 ^~~~~~~~
      |                 RTA_SPACE
cc1: some warnings being treated as errors
make[3]: *** [scripts/Makefile.build:285: /home/marcelo/Git/rtl8188ce-linux-driver/rtl8188ee/pwrseqcmd.o] Error 1
make[2]: *** [Makefile:1875: /home/marcelo/Git/rtl8188ce-linux-driver/rtl8188ee] Error 2
make[2]: Leaving directory '/usr/src/linux-headers-5.15.0-25-generic'
make[1]: *** [Makefile:33: default] Error 2
make[1]: Leaving directory '/home/marcelo/Git/rtl8188ce-linux-driver/rtl8188ee

**Comment by FreedomBen** [comment]
@marceloiury what distro are you on?  and are you using the distro kernel or a custom kernel?

**Comment by marceloiury** [comment]
I am using Ubuntu 22.04. Kernel 5.15. It is a distro kernel

**Comment by VintageSteel** [comment]
- FWIW I'm also having exactly the same issue. I'm on Zorin OS 16.1, which is using 5.13.0-48-generic branch...

**Comment by Dr-Pe** [comment]
Hi, I'm having the same error, using a freshly installed Ubuntu 22.04 which is using a 5.15.0-43-generic kernel.

**Comment by iainism** [comment]
Same error here in Ubuntu Studio 22.04.1 (based on 5.15.0-46-lowlatency)

**Comment by fran6120** [comment]
Same error here in `Kubuntu 22.04.1` using distro kernel `5.15.0-47-generic`

```
/home/javi/rtl8188ce-linux-driver/rtl8188ee/pwrseqcmd.c:70:17: error: implicit declaration of function ‘RT_TRACE’; did you mean ‘RTA_SPACE’? [-Werror=implicit-function-declaration]
   70 |                 RT_TRACE( rtlpriv, COMP_INIT, DBG_TRACE,
      |                 ^~~~~~~~
      |                 RTA_SPACE
cc1: some warnings being treated as errors
make[3]: *** [scripts/Makefile.build:297: /home/javi/rtl8188ce-linux-driver/rtl8188ee/pwrseqcmd.o] Error 1
make[2]: *** [Makefile:1881: /home/javi/rtl8188ce-linux-driver/rtl8188ee] Error 2
make[2]: Leaving directory '/usr/src/linux-headers-5.15.0-47-generic'
make[1]: *** [Makefile:33: default] Error 2
make[1]: Leaving directory '/home/javi/rtl8188ce-linux-driver/rtl8188ee'
make: *** [Makefile:41: all] Error 2
```

**Comment by marceloiury** [comment]
Thank you so much

Em ter., 18 de out. de 2022 18:16, Ben Porter ***@***.***>
escreveu:

**Comment by automainint** [comment]
I had the same compilation error. Fixed it by replacing `RT_TRACE` with `rtl_dbg` in file `rtl8188ee/pwrseqcmd.c`

It seems, this issue still not fixed in branch `generic-5.15.x`
